vtk-cdat/build.sh

```diff
@@ -17,7 +17,7 @@ else
         export LD_LIBRARY_PATH=${PREFIX}/lib:${LIBRARY_PATH}
         export CC="gcc -Wl,-rpath=${PREFIX}/lib"
         export CXX="g++ -Wl,-rpath=${PREFIX}/lib"
-        export LDFLAGS="-lm"
+        export LDFLAGS="-L/usr/lib/x86_64-linux-gnu -lm"
```

Comment:
@danlipsa which library is in this directory that wasn't found otherwise? Should we try to bring it in via conda?